src/metabase/models/table.clj

```diff
@@ -239,14 +237,14 @@
 (defn database
   "Return the `Database` associated with this `Table`."
   [table]
-  (t2/select-one Database :id (:db_id table)))
+  (t2/select-one :model/Database :id (:db_id table)))
 
 ;;; ------------------------------------------------- Serialization -------------------------------------------------
 (defmethod serdes/dependencies "Table" [table]
   [[{:model "Database" :id (:db_id table)}]])
 
 (defmethod serdes/generate-path "Table" [_ table]
-  (let [db-name (t2/select-one-fn :name 'Database :id (:db_id table))]
```

Comment:
Given that `Database` was in scope, I'm wondering why it was quoted before 😕

Reply:
yea I think sometimes you just used to the quick way so you don't check whether Database was required :p